.github/workflows/cmake.yml

```diff
@@ -0,0 +1,55 @@
+name: CMake
+
+on:
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ]
```

Comment:
We should free these variables for any branch

Reply:
done.